CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,53 @@
+# Meetup Code of Conduct
+
+All organizers, attendees, speakers, sponsors and volunteers at Kamloops R
+Meetup events are required to agree to uphold the following code of conduct.
+Kamloops R Meetup event organizers and/or Kamloops Innovation staff will enforce
+this code throughout the event. We are expecting cooperation from all
+participants to help ensure a safe environment.
+
+## The Quick Version
+
+Kamloops R Meetup is dedicated to providing a harassment-free event experience
+for everyone, regardless of gender, gender identity and expression, age, sexual
+orientation, disability, physical appearance, body size, race, ethnicity,
+religion (or lack thereof), or technology choices. We do not tolerate harassment
+of event participants in any form. Sexual language and imagery is not
+appropriate for any event venue, including talks, workshops, parties, Twitter
+and other online media. Event participants violating these rules may be
+sanctioned or expelled from the event at the discretion of the event organizers
+or Kamloops Innovation employees.
+
+## The Less Quick Version
+
+Harassment includes offensive verbal comments related to gender, gender identity
+and expression, age, sexual orientation, disability, physical appearance, body
+size, race, ethnicity, religion, technology choices, sexual images in public
+spaces, deliberate intimidation, stalking, following, harassing photography or
+recording, sustained disruption of talks or other events, inappropriate physical
+contact, and unwelcome sexual attention.
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately.
+
+Sponsors are also subject to the anti-harassment policy. In particular, sponsors
+should not use sexualized images, activities, or other material. Sponsor staff
+(including volunteers) should not use sexualised clothing/uniforms/costumes, or
+otherwise create a sexualized environment.
+
+If a participant engages in harassing behavior, the event organizers or Kamloops
+Innovation staff may take any action they deem appropriate, including warning
+the offender or expulsion from the event with no refund.
+
+If you are being harassed, notice that someone else is being harassed, or have
+any other concerns, please contact a member of the organizing team immediately.
+If you have concerns about a member of the organizing team, please email
+beth@kamloopsinnovation.ca.
```

Comment:
Same here... I don't want to use Beth as a contact if we haven't confirmed it with her first.